README.md

```diff
@@ -58,6 +58,28 @@ The SDK is also available as docker image. To import it into a docker host, run
 docker load -i build/tmp/deploy/images/iot2050/sdk-iot2050-debian-arm64-docker-archive.tar.xz
 ```
 
+## Build qemu image
+
+To boot IOT2050 image from qemu, you need a customized image for proper booting.
+Please use kas menu with the following command and select qemu image for target build
+
+```shell
+./kas-container menu
+```
+
+Then below command can be used to boot qemu image on a platform that
+qemu-system-aarch64 is installed.
+
+```shell
+qemu-system-aarch64 -bios ./firmware.bin -m 2G -netdev user,id=net,hostfwd=tcp:127.0.0.1:22222-:22
```

Comment:
I suspect there is ./firmware.bin after building the QEMU image, is there? Better reference the one in the deploy folder.

Reply:
It will be fixed with upper u-boot bug. Then I will reference the exact path.